Add blog post for latest standard call (#121)
publiccodenet/blog: _posts/2020-11-16-notes-from-community-call-5-november-2020.md

---
title: "Notes from community call - 5 November 2020"
date: 2020-11-16
author: Jan Ainali
type: blogpost

> excerpt: Discussion about challenges in implementing the standard

Reviewer: ```suggestion
excerpt: Discussion about challenges in implementing the Standard
```

categories:
  - Community call
---

# 5 November 2020 Standard for Public Code community call

## Update from the Foundation

* [Version 0.2.0](https://github.com/publiccodenet/standard/releases/tag/0.2.0) of the Standard for Public Code was released!
* We have made an [introduction video to the Standard for Public Code](https://www.youtube.com/watch?v=QWt6vB-cipE).
* We are still working on the [question of translations](https://github.com/publiccodenet/standard/issues/336) of the Standard for Public Code, pursuing the path of official translations, but have nothing new to report.

## Attendees

* [Jan Ainali](https://publiccode.net/team/jan-ainali.html)
* [Alba Roza](https://publiccode.net/team/alba-roza.html)
* [Charlotte Heikendorf](https://os2.eu/bruger/charlotte-heikendorf), OS2

## Notes

> During the call, we discussed different reasons why it sometimes is a challenge implementing the Standard for Public Code. It covered several different aspects relating to governance (Who is making the call to comply with the Standard?), language (Are the decision makers comfortable with adopting a standard only described in English?) and committment (What does it mean for the organisation to commit to comply with the Standard?).

Reviewer: ```suggestion
During the call, we discussed why implementing the Standard for Public Code can sometimes be challenging. We covered several different aspects relating to governance (i.e., who decides to comply with the Standard?), language (i.e., are the decision makers comfortable with adopting a standard only described in English?) and committment (i.e., what does it mean for the organisation to commit to comply with the Standard?).
```
